doc/rados/troubleshooting/troubleshooting-osd.rst

```diff
@@ -232,10 +232,21 @@ Or::
 The best way to deal with a full cluster is to add new ``ceph-osds``, allowing
 the cluster to redistribute data to the newly available storage.
 
-If you cannot start an OSD because it is full, you may delete some data by deleting
-some placement group directories in the full OSD.
+If you cannot add a new OSD, you can do the following:
```

Comment:
i think the delete-some-pg solution still applies. and could you put more context of "If you cannot add a new OSD"? under what circumstances, can we use following steps?